azure-pipelines.yml

```
@@ -7,13 +7,24 @@ trigger:
 - master
 
 pool:
-  vmImage: ubuntu-latest
+  vmImage: macOS-10.14
 
 steps:
-- script: echo Hello, world!
-  displayName: 'Run a one-line script'
+- script: brew install sdl2 qt@5 libslirp libarchive libepoxy
+  displayName: 'Install dependencies'
 
-- script: |
-    echo Add other tasks to build, test, and deploy your project.
-    echo See https://aka.ms/yaml
-  displayName: 'Run a multi-line script'
+- script: mkdir $(Pipeline.Workspace)/build
+  displayName: 'Create build environment'
+
+- script: cmake $(Build.SourcesDirectory) -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DQt5_DIR=$(brew --prefix qt5)/lib/cmake/Qt5
+  displayName: 'Configure'
+  workingDirectory: mkdir $(Pipeline.Workspace)/build
+  
+- script: make -j$(sysctl -n hw.ncpu)
```

`ncpu` is deprecated: https://github.com/ziglang/zig/issues/1252 https://mail.haskell.org/pipermail/ghc-tickets/2013-December/008198.html
```suggestion
- script: make -j$(sysctl -n hw.availcpu)
```

---

thanks, will fix.